WebApiCoreSeed.UnitTests/Services/BaseServiceUnitTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Runtime.CompilerServices;
+using WebApiCoreSeed.Data.EF;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using WebApiCoreSeed.Data.Models;
+using System;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    public class BaseServiceUnitTest

```

Comment:
I'm not a big fan of having base classes with pre-configured state for tests.. this usually leads to coupled tests.

If we switch to xUnit, we can rely on test fixtures.